package.json

```diff
@@ -99,7 +99,7 @@
     "@react-navigation/native": "6.1.12",
     "@react-navigation/stack": "6.3.29",
     "@react-ng/bounds-observer": "^0.2.1",
-    "@rnmapbox/maps": "10.1.20",
+    "@rnmapbox/maps": "10.1.26",
```

Comment:
Note that the RN 0.75 build issues will likely be fixed in `10.1.30`. Let's create an issue to upgrade and remove the patch

Reply:
created https://github.com/Expensify/App/issues/48123

Reply:
I'm going to go through all added patches and create PRs with the library version bumps that should remove them after we fix the current deploy blockers.

Reply:
I will release new version of pager-view today and I will create PR for version bump + removal of patches